qa/tasks/deepsea.py

```diff
@@ -336,20 +329,63 @@ def _introspect_roles(self, deepsea_ctx):
             'dev_env',
             ]
         for var in report_vars:
-            log.info("{} == {}".format(var, deepsea_ctx[var]))
+            self.logger.info("{} == {}".format(var, deepsea_ctx[var]))
+
+    def _log_error_with_anchor(self, message):
```

Comment:
I'm fine with this, but probably here and after it would be bettor to have shorter names, like:
_anchor_error
_anchor_debug
_anchor_warn
_anchor_info

Reply:
What about this?
```
def _anchored(self, message):
   return self.log_anchor + message
def _anchor_info(self, message):
   return self.log.info(self._anchored(message))
def _anchor_debug(self, message):
   return self.log.debug(self._anchored(message))
def _anchor_warn(self, message):
   return self.log.warn(self._anchored(message))
def _anchor_error(self, message):
   return self.log.error(self._anchored(message))
```